app/renderer/ui/components/paymentRequest/index.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react"
+import Popover from "app/renderer/ui/components/popover"
+
+const styles = require("./style.scss")
+
+type status = "expired" | "still into force" | "never funded"
+
+export interface Iprops {
+  creationDate: string
+  expirationDate?: string
+  status: status
+  from: string
+  amount: string
+  actions: Array<(string | { text: any; onClick: any })>
+}
+
+/**
+ * PaymentRequest UI component
+ *
+ * @export
+ * @class PaymentRequest
+ * @extends {React.Component<Iprops>}
+ */
+
+export default class PaymentRequest extends React.Component<Iprops> {
+  // tslint:disable-next-line: completed-docs
+  public render() {
+    const createStatus = (status: status, date: string | undefined) => (
```

Comment:
`createStatus` can be taken out of the `render()` function and likely out of `PaymentRequest` too.